packages/lesswrong/styles/main.scss

```diff
@@ -22,19 +22,8 @@
 @import "admin";
 @import "alignment-forum";
 
-// CSS Modifications for Navbar buttons and styles
-
-.users-menu-contents a {
-  // When the user menu is first opened, it incorrectly detects the first menu
-  // item as hovered, but doesn't activate it like it would with a real hover.
-  // With Bootstrap's default hover effect for <a> links, this would look like
```

Comment:
I'm slightly skeptical about this because my recollection was that the v0 upgrade was what introduced this bug rather than solving it. But if you are confident (and have tested) that it works, cool.

Reply:
Yep, was fixed by the upgrade, tested directly.